dynamic_forms/utils.py

```diff
@@ -92,6 +92,8 @@ def process_field_from_json(field_json):
     if not isinstance(field_json, dict):
         raise TypeError("Each field JSON must be a dictionary")
     field_type = field_json['type']
+    if field_type == 'text':
+        field_type = field_json['subtype']
```

Comment:
```suggestion
        field_type = field_json.get('subtype', 'text')
```

Wouldn't this be a bit more robust?